raycicmd/config.go

```diff
@@ -119,6 +119,12 @@ type config struct {
 	// Optional.
 	AllowTriggerStep bool `yaml:"allow_trigger_step"`
 
+	// DisableParallelism sets if it is to disable parallelism in the pipeline,
+	// default is false.
+	//
+	// Optional.
+	DisableParallelism bool `yaml:"diable_parallelism"`
```

Comment:
maybe change this to max parallelism? and we can set it to 1 (for max=1) and 0 for (no parallelism allowed)?